.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates
+
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: /
+  schedule:
+    interval: monthly
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: monthly
```

Comment:
Was not previously enabled; will produce a steady flow of PRs, though they can be ignored if you prefer. (Note that updating the POM would require a switch to 2.361.x and Java 11.)